keys/employees-keys/README.md

```diff
@@ -116,10 +133,159 @@ sub   4096R/CC8BE1D5 2021-04-02
+sub   rsa4096 2021-04-07 [E] [expires: 2023-04-07]
+sig!         BAA0A4837C891E29 2021-04-07  Your Name <your.name@3mdeb.com>
+
+gpg: 2 good signatures
+```
+
+Signing should look as follows. If fingerprint match then confirm with `y`.
+
+```shell
+$ gpg -u piotr.krol@3mdeb.com --sign-key your.name@3mdeb.com
+
+sec  rsa4096/BAA0A4837C891E29
+     created: 2021-04-07  expires: 2023-04-07  usage: SC  
+     trust: ultimate      validity: ultimate
+ssb  rsa4096/4F81AE572F9EFECA
+     created: 2021-04-07  expires: 2023-04-07  usage: E   
+[ultimate] (1). Your Name <your.name@3mdeb.com>
+
+
+sec  rsa4096/BAA0A4837C891E29
+     created: 2021-04-07  expires: 2023-04-07  usage: SC  
+     trust: ultimate      validity: ultimate
+ Primary key fingerprint: D9DE EABF F447 B80F 7EC0  3A4B BAA0 A483 7C89 1E29
+
+     Your Name <your.name@3mdeb.com>
+
+This key is due to expire on 2023-04-07.
+Are you sure that you want to sign this key with your
+key "Piotr Król <piotr.krol@3mdeb.com>" (B2EE71E967AA9E4C)
+
+Really sign? (y/N) y
+```
+
+Check signatures after signing:
+
+```shell
+$ gpg --check-sigs your.name@3mdeb.com
+gpg: checking the trustdb
+gpg: marginals needed: 3  completes needed: 1  trust model: pgp
+gpg: depth: 0  valid:   7  signed:   6  trust: 0-, 0q, 0n, 0m, 0f, 7u
+gpg: depth: 1  valid:   6  signed:   1  trust: 6-, 0q, 0n, 0m, 0f, 0u
+gpg: next trustdb check due at 2021-11-23
+pub   rsa4096 2021-04-07 [SC] [expires: 2023-04-07]
+      D9DEEABFF447B80F7EC03A4BBAA0A4837C891E29
+uid           [ultimate] Your Name <your.name@3mdeb.com>
+sig!3        BAA0A4837C891E29 2021-04-07  Your Name <your.name@3mdeb.com>
+sig!         B2EE71E967AA9E4C 2021-04-07  Piotr Król <piotr.krol@3mdeb.com>
+sub   rsa4096 2021-04-07 [E] [expires: 2023-04-07]
+sig!         BAA0A4837C891E29 2021-04-07  Your Name <your.name@3mdeb.com>
+
+gpg: 3 good signatures
+```
+
+Please send received key in encrypted email to your.name@3mdeb.com.
+
+```shell
+gpg --armor --output your-name-key-signed.asc --export "YOUR KEY ID HERE"
+```
+
+### Git configuration
+
+Generated key should be used for signing every commit you push on behalf of
+3mdeb. Detailed instruction for git configuration can be found
+[here](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). In short
+you need following in `~/.gitconfig`:
+
+```bash
+[alias]
+        ci = commit -s -S
+[user]
+        name = Your Name
+        email = your.name@3mdeb.com
+        signingkey = 48579AA47429663E
+```
+
+Make sure your
+[Github](https://docs.github.com/en/github/authenticating-to-github/telling-git-about-your-signing-key#telling-git-about-your-gpg-key-2)
+and [Gitlab](https://docs.gitlab.com/ee/user/project/repository/gpg_signed_commits/index.html#adding-a-gpg-key-to-your-account) is set up correctly to provide GPG commit verification.
+
+
+### Create pull request with key
+
+After everything is set up please issue pull request to this repo with signed
```

Comment:
Are these branch instructions at [3mdeb/news-and-ideas](https://github.com/3mdeb/news-and-ideas#add-new-post) repository-specific, or this is a common workflow across all the 3mdeb repos?

Reply:
@mikebdp2 I would not say news-and-ideas branching applies to this repo. A simple pull request from your for of this repo is good enough.